module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIniFactory.php

```diff
@@ -72,8 +75,18 @@ public function __invoke(
         }
         $pathStack = [
             APPLICATION_PATH . '/languages',
-            LOCAL_OVERRIDE_DIR . '/languages',
         ];
+        $dirLocationsStack = $this->getDirLocationsStack();
+
+        $localPathStack = array_map(
+            function ($dirLocations) {
```

Comment:
I only recently learned that PHP supports arrow functions, but this would be a great spot for one, reducing three lines down to:

`fn ($dirLocations) => $dirLocations['dir'] . '/languages',`

Reply:
Also, would it be more clear to use singular $dirLocation since each instance is just one location?

Reply:
Good Idea! Done.